torchvision/prototype/transforms/functional/_type_conversion.py

```diff
@@ -41,5 +41,3 @@ def to_image_tensor(image: Union[torch.Tensor, PIL.Image.Image, np.ndarray]) ->
 # We changed the names to align them with the new naming scheme. Still, `to_pil_image` is
 # prevalent and well understood. Thus, we just alias it without deprecating the old name.
 to_pil_image = to_image_pil
-
-convert_image_dtype = _F.convert_image_dtype
```

Comment:
This was in here by mistake I think. The old transform `ConvertImageDtype` as well as the other conversion transform functionals were already under the `_meta` namespace. Thus, I moved this there.